pypeit/ech_coadd.py

```diff
@@ -0,0 +1,201 @@
+
```

Comment:
Add a comment describing the purpose of this module.

Reply:
done